apps/tools/tests/test_test_course.py

```diff
@@ -0,0 +1,55 @@
+import collections
+import json
+import os
+
+from librelingo_json_export.export import export_course
+from librelingo_yaml_loader.yaml_loader import load_course
+
+Settings = collections.namedtuple(
+    "Settings",
+    [
+        "dry_run",
+    ],
+)
+
+
+def test_course(tmpdir):
+    root = os.path.abspath(__file__)
+    root = os.path.dirname(root)
+    root = os.path.dirname(root)
+    root = os.path.dirname(root)
+    root = os.path.dirname(root)
```

Comment:
is there a different way to get this root path, I'm thinking? and if not, perhaps there's a way to put it into an automatically applied pytest fixture?

Reply:
It disturbs me too that dirname does not accept  a second parameter of how many levels.
I think it could be moved to a fixture only if we knew that every test file that might need this will be at the same depths in our tree.

Reply:
that's a good point, I didn't think about that. then probably this is the best, I mean, the next best thing I could think of would be a reduce function, but I don't like reduce functions so it will be fine like this